docs/reference/generators.rst

```diff
@@ -4,6 +4,18 @@
 Generators and Application Modeling
 ===================================
 
+Constructors
```

Comment:
This one, I think, does make sense to include with the `BQM` class. I know it's a bit inconsistent, but I think that's much closer to its intent.